pkg/controller.v1alpha3/consts/const.go

```diff
@@ -35,6 +38,11 @@ const (
 	// which is used to run healthz check using grpc probe.
 	DefaultGRPCService = "manager.v1alpha3.Suggestion"
 
+	// DefaultGRPCRetryCount is the the maximum number of retries for gRPC calls
```

Comment:
Should we have DefaultGRPCRetryAttempts here?

Reply:
Good catch, thanks. Fixed that.